src/vsphere_cpi/lib/cloud/vsphere/resources/cluster.rb

```diff
@@ -11,6 +11,8 @@ class Cluster
 
       CLUSTER_VM_GROUP_SUFFIX = '_vm_group'.freeze
       CLUSTER_VM_HOST_RULE_SUFFIX = '_rule'.freeze
+      CLUSTER_VM_HOST_RULE_SHOULD = 'SHOULD'.freeze
```

Comment:
Not sure either of these constants are very helpful. They're kind of just longer versions of the actual string.

Reply:
They are used in multiple places so I wanted to keep constants which will avoid spelling mistakes by developers and thus avoid the bugs